src/test/resources/json/WithJointPartyNoSubscription.json

```diff
@@ -0,0 +1,381 @@
+            "hearingLoop"
+          ],
+          "excludeDates": [
+            {
+              "value": {
+                "start": "2018-04-04",
+                "end": "2018-04-06"
+              }
+            },
+            {
+              "value": {
+                "start": "2018-04-10"
+              }
+            }
+          ]
+        },
+        "appealReasons": {
+          "reasons": [
+            {
+              "value": {
+                "reason": "reason1",
+                "description": "description1"
+              }
+            },
+            {
+              "value": {
+                "reason": "reason2",
+                "description": "description2"
+              }
+            }
+          ],
+          "otherReasons": "Another reason"
+        },
+        "rep": {},
+        "signer": "Yes",
+        "supporter": {
+          "contact": {
+            "email": "pacetest@yahoo.com",
+            "mobile": "07985223409"
+          },
+          "name": {
+            "firstName": "Guy",
+            "lastName": "Wilder",
+            "middleName": "Clark Eaton",
+            "title": "Mr"
+          }
+        }
+      },
+      "caseCreated": "2018-02-13",
+      "caseReference": "SC022/14/12423",
+      "subscriptions": {
+        "appellantSubscription": {
+          "email": "tester@hmcts.net",
+          "mobile": "07985233301",
+          "reason": null,
+          "subscribeEmail": "No",
+          "subscribeSms": "No",
+          "wantSmsNotifications" : "No",
+          "tya": "543212345"
+        },
+        "jointPartySubscription": {
+          "email": "jointParty@hmcts.net",
+          "mobile": "07983469702",
+          "reason": null,
+          "subscribeEmail": "Yes",
+          "subscribeSms": "Yes",
+          "wantSmsNotifications" : "Yes",
+          "tya": "232929249492"
+        }
+      },
+      "region": "CARDIFF",
+      "regionalProcessingCenter": {
+        "name": "CARDIFF",
+        "address1": "HM Courts & Tribunals Service",
+        "address2": "Social Security & Child Support Appeals",
+        "address3": "Eastgate House",
+        "address4": "Newport Road",
+        "city": "CARDIFF",
+        "postcode": "CF24 0AB",
+        "phoneNumber": "0300 123 1142",
+        "faxNumber": "0870 739 4438"
+      },
+      "hearings": [{
+        "id": "1234",
+        "value": {
+          "hearingDate": "2018-01-12",
+          "time": "11:00",
+          "venue": {
+            "name": "Prudential House",
+            "address": {
+              "line1": "36 Dale Street",
+              "line2": "",
+              "town": "Liverpool",
+              "county": "Merseyside",
+              "postcode": "L2 5UZ"
+            },
+            "googleMapLink": "https://www.google.com/theAddress"
+          }
+          }
+        }
+      ]
+    },
+    "case_type_id": "Benefit",
+    "state": "appealCreated",
+    "id": "12345656789"
+  },
+  "event_id": "reviewConfidentialityRequest"
```

Comment:
Is this the right event? the json file is a bit of a strange name - doesn't describe the event.
I thought this PR is for joint party added event